pkg/api-server/dataplane_overview_ws.go

```diff
@@ -83,7 +83,7 @@ func (r *overviewWs) inspectDataplanes(request *restful.Request, response *restf
 	tags := parseTags(request.QueryParameters("tag"))
 	gateWayFilterQueryParam := request.QueryParameter("gateway")
 	if gateWayFilterQueryParam == "true" {
-		gatewayFilterTag := map[string]string{"service": "gateway"}
+		gatewayFilterTag := map[string]string{"service": "*"}
```

Comment:
In this case, gateway query param does nothing. Service tag is required on every dataplane gateway or not, so you won't filter anything.

Reply:
I am not getting you on this. Can you elaborate clearly

Reply:
should I need to write a separate method to get only gateway data planes

Reply:
Yes, you should filter by existence of the `gateway` section in networking.